sdk/keyvault/keyvault-admin/generate.mjs

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+
+import "zx/globals";
```

Comment:
Ignore this, just wanted to be explicit on what steps I needed and wont be part of the final PR